ats/atsMachines/fluxScheduled.py

```diff
@@ -0,0 +1,101 @@
+#ATS:slurm56 SELF FluxScheduled 3
+#ATS:toss_4_x86_64_ib SELF FluxScheduled 16
+
+from ats.atsMachines import lcMachines
```

Comment:
💨 🧹  [isort](https://pycqa.github.io/isort/) is an import sorting tool that does this for you - also available as a [vscode extension](https://github.com/microsoft/vscode-isort) until you transition to [neovim](https://github.com/neovim/neovim) :wink:

Reply:
Are flux imports considered third party imports (should they go before ats imports or with ats imports?) Not sure why, but isort is trying to group them like this, which looks silly to me:
```python3
import time
from math import ceil

import flux.job
from ats import terminal
from ats.atsMachines import lcMachines

import flux
```
It turns out, command line isort and vscode isort disagree on where flux should go. CLI says this which looks nicer to me, but still not sure if there should be a line between flux and ats imports:
```python3
import time
from math import ceil

import flux
import flux.job
from ats import terminal
from ats.atsMachines import lcMachines
```

Reply:
Flux would definitely be a third party import in this case. Flux is a separate project from ATS. If ATS was in the flux-framework GitHub org and written by Flux developers it would be different.

So with that in mind I would say it should be 

```python
import time
from math import ceil

import flux
import flux.job

from ats import terminal
from ats.atsMachines import lcMachines
```

i.e. the second one, but with an extra line like you said.